libs/s25main/Loader.cpp

```diff
@@ -569,7 +573,7 @@ void Loader::fillCaches()
                                 id += NATION_RTTR_TO_S2[nation] * 6;
                             } else if(nation == NAT_BABYLONIANS) //-V547
                             {
-                                id += NATION_RTTR_TO_S2[nation] * 6;
+                                id += NATION_RTTR_TO_S2[NAT_ROMANS] * 6;
```

Comment:
this fixes a different bug?

Reply:
Not really. clang-tidy has a new warning "duplicated branches". I just made the "difference" here explicit to silence it.

Reply:
but  `NAT_BABYLONIANS` isnt `NAT_ROMANS`, is it?

Reply:
There is no babylonians in S2, this makes it clear we are reusing the roman figures until actual new ones exist. See the comment below.

Reply:
ok